tareas/Tarea3_Avd/raices.jl

```diff
@@ -97,299 +87,35 @@ function unicity(f, V, tol)
 end
```

Comment:
Dos comentarios sobre esta función, en particular el bloque `for`:
- Si alíneas los `end` con lo que corresponden (`for`s o `if`s en este caso) haces la lectura más sencilla
- Si utilizas broadcasting puedes sencillamente escribir `uni = esmonotona.(f, V)`